src/test/java/dl/news/portal/domain/service/NewsServiceTest.java

```diff
@@ -85,4 +87,11 @@ public void findByUpdatedDateBetween() {
         assertNotEquals(allNews.size(), news.size());
     }
 
+    @Test
+    public void findPageByAuthor() {
```

Comment:
how this test work? What it tests ?

Reply:
Nothing. This test and service method redundant and removed in searching branch